.github/workflows/npmpublish.yml

```diff
@@ -39,35 +39,35 @@ jobs:
           json -I -f package.json -e "this.version=\"$VERSION\""
           json -I -f package-lock.json -e "this.version=\"$VERSION\""
           npm run build
-          ./scripts/docs-config.sh "${VERSION:1}"
+          ./scripts/docs-config.sh "${VERSION}"
           npm run docs
           npm run typedoc
           npm pack
         env:
-          VERSION: ${{ github.event.release.tag_name }}
+          VERSION: ${{ github.event.release.tag_name:1 }}
       - name: Publish to NPM
         run: ./scripts/publish.sh
         env:
           NODE_AUTH_TOKEN: ${{secrets.NPM_TOKEN}}
-          VERSION: ${{ github.event.release.tag_name }}
+          VERSION: ${{ github.event.release.tag_name:1 }}
       # On releases, VERSION is the tag name which is the version
       # However, it will include the leading "v", so we need to strip that
       # first character off here since we want the docs folder to not have
       # the "v" in it.
       - name: Deploy Docs
-        run: ./scripts/deploy-docs.sh "${VERSION:1}"
+        run: ./scripts/deploy-docs.sh "$VERSION"
         env:
           GITHUB_TOKEN: ${{ secrets.GH_AUTH_TOKEN }}
           GH_AUTH_EMAIL: ${{ secrets.GH_AUTH_EMAIL }}
-          VERSION: ${{ github.event.release.tag_name }}
+          VERSION: ${{ github.event.release.tag_name:1 }}
       - name: Upload NPM package file
         id: upload-npm-package-file
         uses: actions/upload-release-asset@v1
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          VERSION: ${{ github.event.release.tag_name }}
+          VERSION: ${{ github.event.release.tag_name:1 }}
         with:
           upload_url: ${{ github.event.release.upload_url }}
-          asset_path: ${{ format('chart.js-{0}.tgz', github.event.release.tag_name) }}
-          asset_name: ${{ format('chart.js-{0}.tgz', github.event.release.tag_name) }}
+          asset_path: ${{ format('chart.js-{0}.tgz', github.event.release.tag_name:1) }}
```

Comment:
Another test for this:
https://github.com/kurkle/action-tests/blob/main/.github/workflows/asset.yml

In there I tested crossing job boundaries with the outputs and cleaner string concatenation